app/Jobs/ExportPostsJob.php

```diff
@@ -88,5 +89,11 @@ public function handle(PostCount $usecase, ExportJobRepository $exportJobRepo)
                 ($batchNumber === 0)
             ));
         }
+
+        // Queue combine job in 1 min
+        dispatch(
+            (new CombineExportedPostBatchesJob($this->jobId))
```

Comment:
so it does the first try after 1 minute, but if not done, it'll just retry until it can do it, right?

Reply:
It'll queue it to run in 1 minute. Then the CombineExportedPostBatchesJob checks if all the batches are done, and if they're not retries in 5 mins.